public/editor/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -15,6 +15,13 @@ define(function(require) {
         // Determine where to horizontally place menu based on button's icon location
         var menuWidth = self._menu$.width();
         var leftOffset = self._button$.offset().left - menuWidth/2 + 11;
+        var iconWidth = $("#filetree-pane-nav-add").width();
+        var arrowOffset = menuWidth/2 - iconWidth/2;
+        if(leftOffset < 0) {
+          arrowOffset = menuWidth/2 - iconWidth/2 + leftOffset;
+          leftOffset = 0;
+        }
+        $("#filetree-pane-nav-arrow-tip").css("left", arrowOffset);
```

Comment:
Here, again - we should not reference the filetree element, but instead find an ``.arrow`` element in the ``$menu`` element. I bet something like ``self._menu$.find('.arrow')`` will work. 

However, since the gear menu doesn't yet have an arrow element as a ``<div>`` you'll also have to add one there.